tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -151,6 +151,38 @@ public function test_are_site_fields_synced() {
 		$this->assertEquals( $failure_response_data, false );
 	}
 
+	public function test_url_handles_staging() {
+		$url = $this->google_proxy->url();
+		$this->assertEquals( $url, Google_Proxy::PRODUCTION_BASE_URL );
+		// The test for this behaviour depends on a constant value which can
+		// only be redefined if PECL extension runkit7 is installed.
+		if ( defined( 'GOOGLESITEKIT_PROXY_URL' ) ) {
+			if ( function_exists( 'runkit7_constant_remove' ) ) {
+				runkit7_constant_remove( 'GOOGLESITEKIT_PROXY_URL' );
+			} else {
+				return;
+			}
+		}
```

Comment:
Let's mark this test skipped if we don't have the `runkit7` extension:
```suggestion
		// The test for this behaviour depends on a constant value which can
		// only be redefined if PECL extension runkit7 is installed.
		if ( ! extension_loaded( 'runkit7' ) ) {
			$this->markTestSkipped( 'The runkit7 extension is not available.' );
		}
        
		if ( defined( 'GOOGLESITEKIT_PROXY_URL' ) ) {
			runkit7_constant_remove( 'GOOGLESITEKIT_PROXY_URL' );
		}
```